### ISSUE TITLE
havenworks.com

### ISSUE THREAD

**Issue by terrorist96** [uBlockOrigin/uAssets#1853]
### URL(s) where the issue occurs

`http://havenworks.com/`

### Describe the issue

Redirects to random sites.
 
### Versions

- Browser/version: Chrome 65
- uBlock Origin version: 1.15.18

### Settings

Default

### Notes

Each time you go to that URL, it redirects somewhere else.

**Comment by zaphodbeebblebrox** [comment]
Did you mean to go to `https://www.havenworks.org`? Havenworks.com is just a parked domain.

**Comment by terrorist96** [comment]
Funnily, one of the random redirects of the `.com` led to the `.org` site. But no, I wasn't attempting to go anywhere. I just saw this site (the `.com`) somewhere and wanted to check it out.

**Comment by okiehsch** [comment]
I get redirected to
````
http://ww1.havenworks.com/?sub1=5b9a6d10-352c-11e8-a387-f7f6790122bd
````
and only to that sedo-site.
Anyway this is a non-issue as havenworks is just a parked domain.

**Comment by terrorist96** [comment]
@okiehsch yeah I get that site too. But other times I get

```
https://rep.gettopappz.com/offer.html?tp=iw&cid=7136&v=52&gnum=6&clickid=00240416p060884491753&cachecode=KrQNmERNMiCfFQMuV4Kzkw&q=havenworks&dkw=havenworks.com&vn=1&extid=gfcldokolejkkpmgplfhljpkdnejpana&yid=38pr&extname=Searching%20Safely&tid=179&vc=8&gr=1&c=00240416p060884491753
```

**Comment by okiehsch** [comment]
I can' reproduce and really it is a **parked** domain, so I fail to see the issue.
I would advise against clicking continue at this `gettopappz` site.

**Comment by WowDude** [comment]
It redirected me to `http://iyfnzgb.com/?pid=9PO1H9V71&dn=havenworks.com` which was blocked by uBO by `hpHosts’ Ad and tracking servers`

**Comment by okiehsch** [comment]
I tried again, I always get redirected to the same sedo parking site.
I will not strict-block a parked domain, if I can't reproduce the issue, if mapx- or gorhill can reproduce they can add the site.

**Comment by WowDude** [comment]
@okiehsch : it seems it's geo-locked, I've tried several proxies from various countries: United Kingdom, Netherlands, United States, Singapoure and indeed it redirects only to the sedo-site, but on some other countires it did 3 chained redirections in a row to scam fake websites, so step to reproduce is to visit the website using use different countries proxies, for example like PL or JP:

On PL (for example `147.135.210.114:54566` ):

1. `http://usa.bravo-dog.com/zcvisitor/01c5a841-364e-11e8-
bc77-0a8ca3fd4c72?campaignid=03bc31b0-26a3-11e8-9b3c-0e9c191f3024`

2. `http://usa.gall-tarquin.com/zcredirect?visitid=01c5a841-364e-11e8-bc77-0a8ca3fd4c72&type=js&browserWidth=&browserHeight=&iframeDetected=false`

3. `http://automoney.qtder.com/?subtracking=5qkd505N5G9b4c68313ecae5b26b2b3888a0f73a8d43d2bc1c`

Another try it did 2 chained redirections:

1. `https://sarah.runtnc.net/tr?id=019bdcba68e7b61e95546b228c971ae17418c57eb5.r&tk=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwdWIiOiI1MjJjNjE1YTlhODQ4MGNhYjhiMTA0MTIiLCJ0cyI6IjA0MDIwODI1IiwiZCI6ImhhdmVud29ya3MuY29tIn0.P1jwBY0WJnlxlbS-T63ffr_utJUMJvq_WDSE_xQMjq8`

2. `https://www.secure-warning.com/c/1/?c=t4&key=ron&country=&scr=57edb4f86cc1b60d323e158c`

And on JP (japan, for example `180.235.40.162:8080`) it did also 3 chained redirections:

1. `https://sarah.runtnc.net/tr?id=01ac23ea886899292e6192f91628480fc64c76ca7c.r&tk=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwdWIiOiI1MjJjNjE1YTlhODQ4MGNhYjhiMTA0MTIiLCJ0cyI6IjA0MDIwODQwIiwiZCI6ImhhdmVud29ya3MuY29tIn0.02BU8e1HSfPYsSxPVMj8SgfN6dAZpnXc1efXImWrwV4`

2. `http://11165150.id.clicksguide.net/redirect?o=75&y=150&x=350&r=https%3A%2F%2Fwww.facebook.com%2Fcampaign%2Flanding.php%3Fcampaign_id%3D545416739143593%26extra_1%3D0145815441054112017416521100227814363533%26placement%3D11165150%26creative%3D320058%26keyword%3D%7BKeyword%7D%26partner_id%3Dadnet%26extra_2%3D%7BTrackingParam%7D&u=0145815441054112017416521100227814363533&a=72&t=4827538&g=340107662622457908~712578270610482996&cb=0&faid=4827538&fint=1&b=f.fUS,f.wF,LWii&epcCD=1522658418057&cc=392&epcRFU=null&tk=&k=&qk=2jYILntaHN&nw=SEARCH&tgt=4827538&tp=11165150-57edb4f86cc1b60d323e158c&vu=null&ir=1&tt=RON&ck=0~0&rk=1&ptt=&f=https%3A%2F%2Fwww.facebook.com%2Fcampaign%2Flanding.php%3Fcampaign_id%3D545416739143593%26extra_1%3D0145815441054112017416521100227814363533%26placement%3D11165150%26creative%3D320058%26keyword%3D%7BKeyword%7D%26partner_id%3Dadnet%26extra_2%3D%7BTrackingParam%7D&sc=null&st=null&id=0&it=0&nbrs=0&nk=4827538&fwc=0&spa=&spt=&spc=`

3. `https://www.facebook.com/campaign/landing.php?campaign_id=545416739143593&extra_1=0145815441054112017416521100227814363533&placement=11165150&creative=320058&keyword={Keyword}&partner_id=adnet&extra_2={TrackingParam}`

**Comment by terrorist96** [comment]
I'm in the US.
